ACKNOWLEDGEMENT.md

```
@@ -0,0 +1,33 @@
+# Third-Party Licenses and Notices
+
+This software packages and uses third-party libraries and resources.
+The original copyrights and licenses are detailed below, their fulltext is in `resources/app/licenses`.
+
+### Icons
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)